src/000-SCRIPT_OBJ/Helpers.js

```diff
@@ -251,18 +251,19 @@ App.PR = new function() {
         this.pEffectMeter = function(effect, item)
         {
             var output = "";
+            var effectStr = effect.replace(/ /g, '&nbsp;');
```

Comment:
Not elegant, but otherwise rendering can split description right here, and it looks terrible in tooltips.